invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -22,10 +21,11 @@
 class MARCXMLSchema(BaseSchema):
     """Schema for records in MARC."""
 
+    doi = fields.Method("get_doi")
+    oai = fields.Method("get_oai")
```

Comment:
Could this be obtained using the `attribute`?
```suggestion
    oai = fields.Str(attibute="pids.oai.identifier")
```